docs/developer-docs/latest/setup-deployment-guides/configurations/required/middlewares.md

```diff
@@ -95,16 +95,17 @@ Strapi's core includes the following internal middlewares, mostly used for perfo
 - [query](#query),
 - [response-time](#response-time),
 - responses, which handle the [responses](/developer-docs/latest/development/backend-customization/requests-responses.md),
-- [security](#security),
 - [public](#public),
+- [security](#security),
+- [session](#session),
 
 ::: caution
```

Comment:
All of the core middlewares listed above are added to a new project, but only the middlewares listed in the caution note have to be kept to not throw an error? Is this correct? Otherwise, the caution message needs to be reworded.

Reply:
It is exactly that :) There are mandatory. Do you want to change the wording?

Reply:
In that case, I think the current version is fine, I just wanted to make sure I understood correctly.